### DISCUSSION TITLE
Export to printable version

### DISCUSSION THREAD

**Discussion by Mateusz2088** [FacilMap/facilmap#155] [Ideas]
Hello.
I am scout from Poland. We often use maps for field games. Participants are given paper maps with marked points on the map and must be guided to the point without using GPS. For younger people we export maps with marked trails, but unfortunately there is no tool to export maps with good quality trails, so we take pictures of the screen, which reduces the quality.
Could you make a map export tool, e.g. to pdf or some other vector graphics file with marked trails?
Me, my team and many scouts would be grateful to you.
Mateusz

**Comment by cdauth** [comment]
Hey,

I think this is a good idea, but I think in the near future there are other issues that have a higher priority for me.

I think two specific improvements would be needed for this:
* Improve the print style of FacilMap. Right now it seems that the map does not fill the whole page, and the search bar and other UI elements that are not relevant for printing are shown.
* Include a vector-based map style into FacilMap. I'm not aware of any good OpenStreetMap-based vector maps right now, but I could do some research.

Do you think these two improvements would be enough to fulfill your use case?

**Reply by Mateusz2088** [reply]
Even if it is not in the vector, I will manage somehow, but the most important thing for me is to export the map view with trails. When I have time, I will think about this problem and test my ideas on how to do it.
PS. On osm.org you can export to SVG.

**Reply by cdauth** [reply]
That's really interesting, I didn't know that osm.org can export as SVG. I will investigate how they do it.

**Comment by cdauth** [comment]
Converted this into issue #253
